example/vanilla-k8s-RWO-Block-Volumes/example-raw-block-static-snapshot.yaml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: snapshot.storage.k8s.io/v1
+kind: VolumeSnapshotContent
+metadata:
+  name: static-raw-block-snapshotcontent
+spec:
+  deletionPolicy: Delete
+  driver: csi.vsphere.vmware.com
+  source:
+    snapshotHandle: b24f15ee-1e7b-491c-b57f-c02b6debe716+58bfb27e-8b04-42a5-8a3a-1a9bcdc92654 # FCD ID + FCD SnapshotID
```

Comment:
Add a comment here asking people to replace with their own FCD and Snapshot IDs.